app/src/main/java/com/github/nitwix/sdpbootcampkotlin/GreetingActivity.kt

```diff
@@ -15,4 +18,9 @@ class GreetingActivity : AppCompatActivity() {
             text = message
         }
     }
+
+    fun easterEgg(view : View){
+        val sb = Snackbar.make(view, "Easter Egg :D", BaseTransientBottomBar.LENGTH_LONG);
```

Comment:
This easter egg doesn't look very professional...

```suggestion
        val sb = Snackbar.make(view, "Professional looking easter egg", BaseTransientBottomBar.LENGTH_LONG);
```